Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04 as base
+
+MAINTAINER FilipS (filips@vaskir.co)
+
+#Disable interactive tzdata instalaltion
+ARG DEBIAN_FRONTEND=noninteractive
+
+
+RUN apt update && \
+    apt install -y make texlive-full latex2html texlive-latex-extra texlive-pstricks texlive-lang-* texlive-xetex
+
```

Comment:
as ```texlive-full``` contains it, you can drop
```texlive-latex-extra```
```texlive-lang-*```
```texlive-pstricks```
```texlive-xetex```

https://packages.ubuntu.com/bionic/texlive-full

Reply:
Dropped the packages, forgot to remove them :)